go.mod

```diff
@@ -3,12 +3,18 @@ module sigs.k8s.io/kubebuilder/v3
 go 1.15
 
 require (
+	github.com/cloudflare/cfssl v1.5.0
 	github.com/gobuffalo/flect v0.2.2
+	github.com/markbates/pkger v0.17.1
 	github.com/onsi/ginkgo v1.12.0
 	github.com/onsi/gomega v1.9.0
 	github.com/spf13/afero v1.2.2
-	github.com/spf13/cobra v0.0.7
+	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5
-	golang.org/x/tools v0.0.0-20200403190813-44a64ad78b9b
+	golang.org/x/tools v0.0.0-20190920225731-5eefd052ad72
+	k8s.io/api v0.20.2 // indirect
```

Comment:
```suggestion
```

Reply:
It shows not used. Could you please remove and run go mod tidy after that to ensure?

Reply:
I've had a lot of trouble with `go.mod` because the latest `ginkgo` / `gomega` break our tests, and `go mod` keeps trying to update them.  I'll try again.

Reply:
#2039 should have fixed that

Reply:
Thanks!